folly/futures/Future-inl.h

```diff
@@ -28,7 +28,7 @@
 #include <folly/futures/detail/Core.h>
 #include <folly/futures/Timekeeper.h>
 
-#if FOLLY_MOBILE || defined(__APPLE__)
+#if !HAVE_BOOST_CONTEXT || (FOLLY_MOBILE || defined(__APPLE__))
```

Comment:
This needs to also adjust the configure check for `HAVE_BOOST_CONTEXT` to only define it if it's <= v1.60